plugins/source/tailscale/resources/plugin/tables.go

```diff
@@ -14,7 +14,9 @@ func tables() []*schema.Table {
 	return []*schema.Table{
 		acl.Acls(),
 		device.Devices(),
-		dns.Dns(),
+		dns.Nameservers(),
```

Comment:
Worth removing `// Code generated by codegen; DO NOT EDIT.` from the file too I think